### PR TITLE
Kristinehamn, Sweden

### PR DESCRIPTION
**Pull request by sergiyprotsiv**
This one has the regex issue to extract street and house number. Help appreciated.

### CODE REVIEW

File: sources/se/municipality_of_kristinehamn.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        "city": "Kristinehamn"
+    },
+    "data": "http://kartor-va.kristinehamn.se/arcgis/rest/services/Bakgrundskartor/Bakgrundskarta_fargad_enkel/MapServer/112",
+    "type": "ESRI",
+    "conform": {
+        "type": "geojson",
+        "number": {
+            "function": "regexp",
+            "field": "Name",
+            "pattern": "([0-9]+([A-Z])?)$"
```
Comment (thatdatabaseguy) [inline sources/se/municipality_of_kristinehamn.json]: Might be an issue with the inner capture group. Also, shouldn't this be identical to the regex below just with different captures? I'd try:  "^(?:.*?)(?: ([0-9]+[A-Z]?))?$"

File: sources/se/municipality_of_kristinehamn.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        },
+        "street": {
+            "function": "regexp",
+            "field": "Name",
+            "pattern": "^(.*?)(?: [0-9]+([A-Z])?)?$",
```
Comment (thatdatabaseguy) [inline sources/se/municipality_of_kristinehamn.json]: Same deal here: "^(.*?)(?: [0-9]+[A-Z]?)?$" should work.

File: sources/se/municipality_of_kristinehamn.json [hunk 3]
```diff
@@ -10,12 +10,12 @@
         "number": {
             "function": "regexp",
             "field": "Name",
-            "pattern": "([0-9]+([A-Z])?)$"
+            "pattern": "^(?:.*?)(?: ([0-9]+[A-Z]?))?$"
```
Comment (thatdatabaseguy) [inline sources/se/municipality_of_kristinehamn.json]: Oh, maybe the captured group shouldn't be optional so it won't return None: "^(?:.*?)(?: ([0-9]+[A-Z]?))$"

**Comment by sergiyprotsiv** [comment]
Great, thanks, that worked!

**Pull request merged**